<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>181</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="hostLabel">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="hostLineEdit">
       <property name="text">
        <string>localhost</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="userLabel">
       <property name="text">
        <string>User</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="userLineEdit">
       <property name="text">
        <string>SYSDBA</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="dialectLabel">
       <property name="text">
        <string>Dialect</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="dialectComboBox">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="charsetLabel">
       <property name="text">
        <string>Charset</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="charsetComboBox">
       <item>
        <property name="text">
         <string>NONE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UNICODE_FSS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ASCII</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTF8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISO8859_2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISO8859_3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISO8859_4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISO8859_5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISO8859_6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISO8859_7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISO8859_8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISO8859_9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ISO8859_13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WIN1250</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WIN1251</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WIN1252</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WIN1253</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WIN1254</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WIN1255</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WIN1256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WIN1257</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WIN1258</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="filenameLabel">
       <property name="text">
        <string>Filename</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="filenameLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pbChoosefile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose file...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbConnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QTableView" name="tvTables">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>541</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tvFields">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>541</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRead">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Tables</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbgenSetter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Setter</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbGenGetter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Getter</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbQObject">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>QObject for Qt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbGenerateHeader">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Header</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbGenerateSource">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>430</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Source</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edtGentext">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>541</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbwxObject">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>wxObject for wxWidgets</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
